regression-test/data/auth_call/multi_load_cm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
what's this?

Reply:
The case will write data into this file and then execute. So this file is temporarily empty.

Reply:
The empty file has been deleted and generated every time it is executed.